### DISCUSSION TITLE
Deactivation of my studen pack

### DISCUSSION THREAD

**Discussion by Tresor-Kasenda** [community/community#113483] [GitHub Education]
### Hi everyone,

I hope you're well. I'm writing to you because I recently encountered a problem with my GitHub Student Pack.

I've been deactivated from my Student Pack, even though I use my university email address for my GitHub account. This means that I no longer have access to the benefits of the Student Pack, in particular the Pro account that was included.

However, I'm still a student at L'Ecole Superieur des Ingenieur Salama and I still meet the eligibility criteria for the GitHub Student Pack. So I'm wondering if there was a misunderstanding or error in the process of deactivating my account.

I would like to stress the importance of being able to benefit from the features offered by the GitHub Student Pack as part of my studies and academic projects. It helps me enormously in my learning and professional development.

Would it be possible to reactivate my Student Pack or to help me solve this problem? I am willing to provide any additional information needed to confirm my student status and regain access to my Pro account.

**Comment by samus-aran** [comment]
Hi @Tresor-Kasenda  👋

Occasionally it is necessary for us to ask you to reverify your current academic status in order to continue receiving your GitHub Education benefits. We may require this when seemingly-legitimate applicants have submitted academic affiliation proof which has been discovered to be used by bad actors as well.

We apologize for any inconvenience this may cause.

To reverify, return to the GitHub Education benefits page( https://education.github.com/benefits ) and click "submit your information again."

Thank you for your understanding.
